state/migration_import_test.go

```diff
@@ -1057,6 +1057,7 @@ func (s *MigrationImportSuite) TestSpaces(c *gc.C) {
 	imported, err := newSt.Space(space.Name())
 	c.Assert(err, jc.ErrorIsNil)
 
+	c.Assert(imported.Id(), gc.Equals, space.Id())
```

Comment:
Suggestion: add a test for spaceID not existing?

Reply:
Done.